editor/svgcanvas.js

```diff
@@ -5083,7 +5083,8 @@ this.importSvgString = function(xmlString) {
 		return false;
 	}
 
-	return true;
+	// we want to return the element so we can automatically select it
```

Comment:
Please update the docs for this function (which still say that it returns true / false)